src/components/authorPageComponents/Map/Map.js

```diff
@@ -1,16 +1,25 @@
-import React from 'react'
+import React from "react"
+import { YMaps, Map, Placemark } from "react-yandex-maps"
 
-import style from './Map.module.scss';
+const MyMapComponent = ({
+  mapState,
+  markGeometry,
+  ...rest
+}) => (
+  <YMaps>
+    <Map state={mapState} {...rest}>
+      {markGeometry.map(coordinate => (
+        <Placemark
```

Comment:
add key for Placemark like `key={coordinate.Latitude+coordinate.Longitude}` or another uniq key-gen